src/components/tuTable/tuTableStore.ts

```diff
@@ -149,13 +164,19 @@ export class TuTableStore {
 				// apply search filter
 				data = _.filter(this.table.data, (value: TuTableRow) => {
 					return _.every(this.table.currentFilters, (filterValue) => {
-						const fieldValue = this.getField(value.rowData, filterValue.field);
-						if (filterValue.value === "")
-							return true;
-						else if (fieldValue !== undefined)
-							return String(fieldValue).toLowerCase().includes(String(filterValue.value).toLowerCase());
-						else
-							return true;
+						const fieldValue = this.getField(
+							value.rowData,
+							filterValue.field
+						);
+						if (filterValue.value === "") return true;
+						else if (fieldValue !== undefined) {
```

Comment:
You dont need `else` if the first statement is doing a return, just use `if`

Reply:
Okay